enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/InteractiveQuestion.unit.spec.tsx

```diff
@@ -116,7 +123,7 @@ describe("InteractiveQuestion", () => {
   });
 
   it("should render loading state when rerunning the query", async () => {
-    setup();
+    setup({ withCustomLayout: true });
```

Comment:
Only use the custom layout for this test (which needs the "Run Query" button), otherwise use the default question component's layout for increased reliability.